esp-hal/src/uart/mod.rs

```diff
@@ -855,6 +860,35 @@ where
         while !self.is_tx_idle() {}
     }
 
+    /// Sends a break signal for a specified duration in bit time, i.e. the time
+    /// it takes to transfer one bit at the current baud rate. The delay during
+    /// the break is just is busy-waiting.
+    pub fn send_break(&mut self, bits: u32) {
```

Comment:
```suggestion
    #[instability::unstable]
    pub fn send_break(&mut self, bits: u32) {
```

Reply:
The doc comment should follow established style: one line short summary, separate description.